deployment/common/proposalutils/mcms_helpers.go

```diff
@@ -248,3 +256,18 @@ func BatchOperationForChain(
 		Transactions:  []mcmstypes.Transaction{tx},
 	}, nil
 }
+
+func GetAptosRoleFromAction(action mcmstypes.TimelockAction) (mcmsaptossdk.TimelockRole, error) {
```

Comment:
can this be private ?
```suggestion
func getAptosRoleFromAction(action mcmstypes.TimelockAction) (mcmsaptossdk.TimelockRole, error) {
```

Reply:
Also using it in `deployment/ccip/changeset/aptos/utils/mcms.go`. This could go to Aptos mcms sdk in the future, tho. Let me add this comment.